app/db/init_data.py

```diff
@@ -4,6 +4,7 @@
 import logging
 from datetime import datetime
 from functools import reduce
+from pprint import pprint
```

Comment:
Looks like this is not used.

Reply:
Ohh, yeah. I totally forgot to remove this.